src/docs/asciidoc/deployment/configuration/web-ui_configuration.adoc

```diff
@@ -91,6 +91,9 @@ a|card time display mode in the feed. Values :
 |feed.geomap.maxZoom|11|no| Max zoom level, to prevent zooming in too much when only one card is shown (or multiple cards in close proximity).
 |feed.geomap.zoomDuration|500|no| Time in milliseconds it takes to zoom the map to the specific location. Set to 0 to disable the zoom animation.
 |feed.geomap.enableGraph|false|no|Show a graph on the map with number of cards per severity.
+|feed.geomap.bglayer.xyz.url||no|Custom xyz/tiled background layer service URL, Replaces OSM background layer (Add endpoint with {x}{y}{z} variables. Example: "https://service.pdok.nl/brt/achtergrondkaart/wmts/v2_0/grijs/EPSG:3857/{z}/{x}/{y}.png".
+|feed.geomap.bglayer.xyz.tileSize||no|Custom xyz/tiled background layer tile size (Int value, example: 256. Required when using custom background layer).
+|feed.geomap.bglayer.xyz.crossOrigin|null|Custom xyz/tiled background layer crossOrigin setting.
```

Comment:
Modification needed (in bold) : 

|feed.geomap.bglayer.xyz.url|**null**|no|Custom xyz/tiled background layer service URL, Replaces OSM background layer (Add endpoint with {x}{y}{z} variables. Example: "https://service.pdok.nl/brt/achtergrondkaart/wmts/v2_0/grijs/EPSG:3857/{z}/{x}/{y}.png".
|feed.geomap.bglayer.xyz.tileSize|**null**|no|Custom xyz/tiled background layer tile size (Int value, example: 256. Required when using custom background layer).
|feed.geomap.bglayer.xyz.crossOrigin|null|**no**|Custom xyz/tiled background layer crossOrigin setting.

Reply:
Changed the docs to your recommendation.